pkg/virt-operator/util/config_test.go

```diff
@@ -176,6 +177,14 @@ var _ = Describe("Operator Config", func() {
 				{Name: key2, Value: val2},
 			}
 
+			sortEnvVars := func(envVars []k8sv1.EnvVar) {
+				sort.Slice(
+					envVars,
+					func(i, j int) bool { return envVars[i].Name > envVars[j].Name },
+				)
+			}
+			sortEnvVars(*envObjects)
+			sortEnvVars(expected)
 			Expect(reflect.DeepEqual(*envObjects, expected)).To(BeTrue())
```

Comment:
Can you please use `Expect(envVars).To(ConsistOf(expected))`? That will do the right thing.

Reply:
Will it do a "deep" equality?

I will argue that that the source of the problem is the use of `rand` in the first place here. That is not recommended in tests.

Reply:
Done

Reply:
Yes it will: https://onsi.github.io/gomega/ (you have to search for ConsistOf there, can't find anchors).